docs/buildah-commit.md

```diff
@@ -39,13 +51,21 @@ Default leaves the container and its content in place.
 
 ## EXAMPLE
 
-buildah commit containerID
+This example saves an image based on the container.
+ `buildah commit containerID`
+
+This example saves an image named newImageName based on the container.
+ `buildah commit --rm containerID newImageName`
+
 
-buildah commit --rm containerID newImageName
+This example saves an image based on the container disabling compression.
+ `buildah commit --disable-compression containerID`
 
-buildah commit --disable-compression --signature-policy '/etc/containers/policy.json' containerID
+This example saves an image named newImageName based on the container disabling compression.
+ `buildah commit --disable-compression containerID newImageName`
 
-buildah commit --disable-compression --signature-policy '/etc/containers/policy.json' containerID newImageName
+This example commits the container to the image on the local registry using credentials and certificates for authentication.
+ `buildah commit --cert-dir ~/auth  --tls-verify true --creds=username:password containerID docker://localhost:5000/imageId`
```

Comment:
nit: A example where you turn tls-verify to false, would be more likely, since the default is on.

Reply:
Good point, adding.